README.md

```diff
@@ -10,7 +10,7 @@ We teach:
 * Javascript and Jquery
 * How to go about getting a job in the industry
 
-This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
+This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](http://theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
```

Comment:
Actually @codyloyd https on the root domain times out. The current link (http://theodinproject.com) redirects to https://www.theodinproject.com 

I'll change the link to https://www.theodinproject.com